doc/administrator/mapfile.rst

```diff
@@ -83,8 +83,11 @@ example::
 
 ``gml_types``
 
-  Always set this to ``auto``. This means that the layer's field type
-  information is obtained from the input feature driver (OGC).
+  Set this to ``auto``. This means that the layer's field type information is obtained from
+  the input feature driver (OGC). If the type is not well interpreted (notably if you have a
+  wrong operator within your filter), you can define manually your feature type
+  with ``"gml_[feature]_type" "[type]"`` where type is one of
```

Comment:
with ``"gml_<attribute>_type" "<type>"`` where `<type>` is one of

Reply:
I've taken the same notation as MapServer, and as above in the same page. I'll adapt the whole page

Reply:
Thanks :-)